bag/datasets/brk/serializers.py

```diff
@@ -426,7 +434,9 @@ def to_representation(self, instance):
         request = self.context['request']
         subject_natuurlijk = instance.type == instance.SUBJECT_TYPE_NATUURLIJK
 
-        authorized = request.is_authorized_for(authorization_levels.LEVEL_EMPLOYEE_PLUS) or request.user.has_perm('brk.view_sensitive_details')
+        authorized = (request.is_authorized_for(
+            authorization_levels.LEVEL_EMPLOYEE_PLUS) or
+            request.user.has_perm('brk.view_sensitive_details'))
```

Comment:
Deze indent is niet hetzelfde als die op regel 176. Zie ook bijbehorend comment.